django_pony_express/services/base.py

```diff
@@ -230,6 +230,19 @@ def _add_attachments(self, msg: EmailMultiAlternatives):
 
         return msg
 
+    def generate_html_content(self, mail_attributes: dict) -> str:
```

Comment:
thought: LGTM! Maybe this could be a protected method (starting with an underscore), to be consistent with the other non-public methods?

Reply:
Done.